packages/theme/css/foundations/spacing.css

```diff
@@ -0,0 +1,30 @@
+.salt-density-touch {
+    --salt-size-spacing-100: 16px;
```

Comment:
This means it's no longer as easy to change the base spacing value. Is this intended?